src/docfx/build/Build.cs

```diff
@@ -77,56 +88,67 @@ bool ShouldBuildFile(Document file)
         {
             try
             {
+                var model = (object)null;
```

Comment:
>var model = (object)null; [](start = 16, length = 25)

how about "PageModel model = null;"?

Reply:
is may not be a `PageModel`, build TOC returns a `TocModel`

Reply:
`(object)null` is just to keep `var` visually aligned

Reply:
I see

---
In reply to: [202227129](https://github.com/dotnet/docfx/pull/3064#discussion_r202227129) [](ancestors = 202227129)